### PR TITLE
[tech debt] group related classes together

### PR DESCRIPTION
**Pull request by marcparadise**
### Description
This is a re-submit of #63, which was merged prematurely.

### CODE REVIEW

**Comment by marcparadise** [comment]
For those who haven't reviewed yet - when looking by commit, it's a lot cleaner - it shows that the bulk of the change is rename/renamespace/whitespace, eg https://github.com/chef/chef-apply/pull/65/commits/633d26c3578f5e25a35dcb87f7cad8c583a5dade?utf8=%E2%9C%93&diff=unified&w=1

Viewing the full PR  diff hides all of the renames that happened in lib/ and treats them like deleted/new files.

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -20,6 +20,46 @@
 require "rspec/expectations"
 require "support/matchers/output_to_terminal"
 
+# class << Kernel
```
Comment (tyler-ball) [inline spec/spec_helper.rb]: I'm guessing this doesn't need to be in here?

**Comment by marcparadise** [comment]
I'll bring this up to date so we can merge it.

**Comment by marcparadise** [comment]
This is up to date, will merge after verifying recent commits from master are correct post-rebase.

**Pull request merged**